test/lsp/incremental-lsp-changes/incremental-lsp-changes.rec

```diff
@@ -5,7 +5,7 @@ Read: {"jsonrpc":"2.0","method":"workspace/didChangeConfiguration","params":{"se
 Read: {"jsonrpc":"2.0","method":"textDocument/didOpen","params":{"textDocument":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","languageId":"ruby","version":1,"text":"# typed: true\nclass Foo\n    def bar\n        Foo\n    end\nend"}}}
 Read: {"jsonrpc":"2.0","method":"textDocument/didChange","params":{"textDocument":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","version":2},"contentChanges":[{"range":{"start":{"line":3,"character":11},"end":{"line":3,"character":11}},"rangeLength":0,"text":" "}]}}
 Read: {"jsonrpc":"2.0","method":"textDocument/didChange","params":{"textDocument":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","version":3},"contentChanges":[{"range":{"start":{"line":3,"character":12},"end":{"line":3,"character":12}},"rangeLength":0,"text":"+"}]}}
-Write: {"jsonrpc":"2.0","method":"textDocument/publishDiagnostics","params":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","diagnostics":[{"range":{"start":{"line":4,"character":4},"end":{"line":4,"character":7}},"severity":1,"code":2001,"message":"unexpected token kEND","relatedInformation":[{"location":{"uri":"https://srb.help/2001","range":{"start":{"line":0,"character":0},"end":{"line":0,"character":0}}},"message":"Click for more information on this error."}]}]}}
+Write: {"jsonrpc":"2.0","method":"textDocument/publishDiagnostics","params":{"uri":"file:///Users/dmitry/stripe/pay-server/api/lib/compatibility/change/foo.rb","diagnostics":[{"range":{"start":{"line":4,"character":4},"end":{"line":4,"character":7}},"severity":1,"code":2001,"message":"unexpected token kEND","relatedInformation":[]}]}}
```

Comment:
Fixing the relatedInformation bug I pointed out above should fix these recorded tests.